features/step_definitions/stepdefs.js

```diff
@@ -242,15 +253,36 @@ BeforeAll(async function () {
 });
 
 BeforeAll(async function () {
-    captain = new WireMock('http://wiremock:8080');
+    externalActivityPubCaptain = new WireMock('http://fake-external-activitypub');
+    ghostActivityPubCaptain = new WireMock('http://fake-ghost-activitypub');
```

Comment:
Not a blocker - How would you feel about dropping the `Captain` suffix

```
ghostActivityPub.register
```

```
externalActivityPub.register
```

Feels nice